server/routes/index.js

```diff
@@ -1,7 +1,13 @@
+import { Router } from 'express';
+
 import userRouter from './users';
 import articleRouter from './article';
 
-export default (app) => {
-  app.use('/api/v1', userRouter);
-  app.use('/api/v1/articles', articleRouter);
-};
+const router = Router();
+
+router.use(
```

Comment:
This wouldnt scale well when the routes get large. Looking at your app.js file, we would have to use a lot of ```app.use('/api/v1/....', someRouter)``` for different routes. Take a look at #19. This does all the dirty work in index and exports a function which wraps the express app in the app.js file.

Reply:
@ProfJigsaw Not so at all. You would not need to add extra `/api/v1/`.  This is what you have in index.js
```
router.use(
  userRouter,
  articleRouter
);
```
So the first step is to export the router from the different files. You import them in the index.js file and add it to the `router.use` function above. The router is exported and imported in the app.js file.  In the app.js file, you just write `app.use('/api/v1/', router)` and that's it.

Reply:
noted